cmnSys/responseCode.py

```diff
@@ -7,3 +7,5 @@ class ResponseCode(IntEnum):
     UNEXPECTED = 417
     NOT_FOUND = 404
     FORBIDDEN = 403
+    FORCED = 405
+    FORCE_FAILED = 406
```

Comment:
I'll look into similar response codes to use from HTTP to loot for my own purposes